docs/readthedocs/source/_toc.yml

```diff
@@ -81,6 +81,7 @@ subtrees:
               title: "Key Features"
               subtrees:
                 - entries:
+                  - file: doc/Nano/Overview/cuda_patch
```

Comment:
I am not very sure whether we should put this CUDA patch as the first feature? Is it a feature for PyTorch only? Should we put it after PyTorch train & PyTorch inference, and also add "PyTorch" in its title somehow?

Reply:
in fact, this cuda patch should be included in torch patch, I think torch patch doesn't belong to PyTorch Train or PyTorch inference, but we can simply leave this issue to junwei

Reply:
I mean maybe it would be better to be put here：
![image](https://user-images.githubusercontent.com/54161268/209598004-7c2f892c-7668-4577-aec9-4d7302f4e445.png)
with the title "PyTorch CUDA patch". As I feel like it is not as important as PyTorch train and PyTorch inference.

At least one thing I think we should change is to make its title in the table of contents on the left side to be "PyTorch CUDA patch" instead of "CUDA patch". We could do it either through change the title in the md file, or add an extra title in the _toc.yml like this
```rst
- file: doc/Nano/Overview/cuda_patch
  title: "PyTorch CUDA Patch"
```

Reply:
changed it